packages/metal-dom/src/dom.js

```diff
@@ -254,6 +254,31 @@ class dom {
 	}
 
 	/**
+	 * Verifies if the element is able to trigger the Click event,
+	 * simulating browsers behaviour, avoiding event listeners to be called by dom.triggerEvent method.
+	 * @param {Element} node Element to be checked.
+	 * @param {!String} eventName The event name.
```

Comment:
This should be `{string}` (lowercase for primitive types, and no **!** since primitive types are already non null by default).